tests/flow/tests.sh

```diff
@@ -0,0 +1,268 @@
+#!/bin/bash
```

Comment:
Why do we need this new file? what's wrong with the current method ?

Reply:
We're moving to a scheme that standardizes driving of flow tests, making it more robust and easy to maintain across modules (it appears that there is a common pattern). It makes it easier to interface RLTest, collect logs, handle valgrind, etc.
We're not using exactly the same script, as the process of convergence is still ongoing and I'm still learning the patterns, but the overall result seems promissing.

Reply:
Note that this change initiated in the RLEC PR, and was also introduced here to support the log games.